### PR TITLE
ToDo List Webpack and structure

### PR DESCRIPTION
**Pull request by kimothokamau**
This pull request is a request to review webpack and initial structure milestone

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup ESLint
+        run: |
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
+{"mode":"full","isActive":false}
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml]: - Kindly make sure that there are no linter errors.
- Check if your linters are running correctly into your branch.

File: package.json [hunk 2]
```diff
@@ -4,22 +4,33 @@
...
+    ,
+    "style-loader": "^3.1.0"
+    ,
+    "webpack": "^5.44.0"
+    ,
     "webpack-cli": "^4.7.2"
+    ,
+    "webpack-dev-server": "^3.11.2"
   }
 }
```
Comment (carlos-ssh) [inline package.json]: - Kindly fix this error when i try to run your project locally.
- Following the steps that you give in your README file.
  - run ``npm i ``
  - run ``npm run build ``
Screen Shot 2021-07-14 at 06 40 24

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @kimothokamau, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kimothokamau** [comment]
a

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._
- [ ] As requested please ensure GitHub actions are running and are green.


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```
Comment (mikenath223) [inline .github/workflows/linters.yml:7]: - [ ] As requested previously please ensure GitHub actions are running and are all green before requesting a re-review.

**Comment by kimothokamau** [comment]
Fixed issue in lintersyml file that was causing github actions not to run

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**